otter_buddy/cogs/interview_match.py

```diff
@@ -121,14 +132,38 @@ async def send_pair_message(self, otter_one: discord.User, otter_two: discord.Us
             conn = EmailConn()
             conn.send_mail(email, "Interview buddy", message)
     
-    def make_pairs(self, week_otter_pool: list, placeholder: discord.Member):
+    def make_pairs(self, week_otter_pool: list, placeholder: discord.Member) -> List[Tuple[discord.Member, discord.Member]]:
         week_otter_pairs: list = []
+        week_otter_collaborator: list = []
+        week_otter_member: list = []
         if len(week_otter_pool) % 2 == 1:
             week_otter_pool.append(placeholder)
-        
-        random.shuffle(week_otter_pool)
-        for idx in range(len(week_otter_pool) // 2):
-            week_otter_pairs.append((week_otter_pool[idx*2], week_otter_pool[idx*2+1]))
+
+        for otter in week_otter_pool:
+            otter_roles = [str(otter_role) for otter_role in otter.roles]
+            if OTTER_MODERATOR in otter_roles or OTTER_ADMIN in otter_roles:
+                week_otter_collaborator.append(otter)
+            else:
+                week_otter_member.append(otter)
+
+        random.shuffle(week_otter_collaborator)
+        random.shuffle(week_otter_member)
+
+        # Pair interviewees with colaborators
+        collaborator_index = 0
+        member_index = 0
+        while collaborator_index < len(week_otter_collaborator) and member_index < len(week_otter_member):
+            week_otter_pairs.append((week_otter_collaborator[collaborator_index], week_otter_member[member_index]))
+            collaborator_index += 1
+            member_index += 1
+
+        # Pair aditional colaborators
+        for idx in range(collaborator_index, len(week_otter_collaborator), 2):
```

Comment:
Nice change, I didn't know about the 3rd parameter

Reply:
Thanks you :)